ocrd_anybaseocr/cli/ocrd_anybaseocr_dewarp.py

```
@@ -77,21 +77,20 @@ def prepare_data(self, opt, page_img, path):
         
     def process(self):
         try:
-            self.page_grp, self.image_grp = self.output_file_grp.split(',')
+            page_grp, self.image_grp = self.output_file_grp.split(',')
         except ValueError:
-            self.page_grp = self.output_file_grp
+            page_grp = self.output_file_grp
             self.image_grp = FALLBACK_IMAGE_GRP
             LOG.info("No output file group for images specified, falling back to '%s'", FALLBACK_IMAGE_GRP)
         if not torch.cuda.is_available():
             LOG.error("Your system has no CUDA installed. No GPU detected.")
             sys.exit(1)
 
         path = self.parameter['pix2pixHD']
-
         if not Path(path).is_dir():
             LOG.error("""\
                 NVIDIA's pix2pixHD was not found at '%s'. Make sure the `pix2pixHD` parameter 
-                in params.json points to the local path to the cloned pix2pixHD repository.
+                in ocrd-tools.json points to the local path to the cloned pix2pixHD repository.
 
                 pix2pixHD can be downloaded from https://github.com/NVIDIA/pix2pixHD
                 """ % path)
```

This should be LOG.raise() instead of LOG.error()+sys.exit().

Besides, it is unacceptable to exit here when CUDA cannot be found. Torch is perfectly capable of running on CPU, and GPU should never be a hard requirement for OCR-D.

---

I am unable to use/find a LOG.raise() function. Could you please give me a link to where this has been defined or used?

---

Sorry, I meant `LOG.critical()` and `raise Exception()`. But `LOG.error()` and `sys.exit(1)` is probably okay as well.